src/app/shared/pipes/date-with-timezone.pipe.ts

```
@@ -0,0 +1,26 @@
+import { BehaviorSubject } from 'rxjs';
+
+dayjs.extend(utc);
+dayjs.extend(timezone);
+
+@Pipe({
+  name: 'dateWithTimezone',
+})
+export class DateWithTimezonePipe implements PipeTransform {
+  constructor(@Inject(TIMEZONE) private timezone$: BehaviorSubject<string>) {}
+
+  transform(value: string | Date, format: string = 'MMM DD, YYYY'): string {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)